docs/docs/guides/integrate/identity-providers/introduction.md

```diff
@@ -151,7 +122,15 @@ When configuring external IdP templates in ZITADEL, several common settings enab
 
 - **Account linking allowed**: Enables existing ZITADEL accounts to be linked with identities from external IdPs. It requires that a linkable ZITADEL account already exists for the user attempting to log in with an external IdP. Account linking is beneficial for users who wish to associate multiple login methods with their ZITADEL account, providing flexibility and convenience in how they access your application.
 
+:::info
 
+If you use the Generic OIDC provider or the Generic OAuth provider note that the use of [Proof Key for Code Exchange](https://datatracker.ietf.org/doc/html/rfc7636) for authorization requests is enabled by default to mitigate the authorization code interception attack.
+
+This setting can be disabled in the Optional settings section when you create or update these providers settings:
+
+![Identity Provider Overview](/img/guides/zitadel_idp_pkce.png)
+
+:::
```

Comment:
i see the intention here, but i'm unsure how many IdPs actually support it and whether we should activate it by default

Reply:
So @livio-a should I set the use_pkce value to false by default, and let users enable it for all cases? So I should change the console accordingly